src/registriesView.ts

```diff
@@ -43,14 +41,8 @@ export class ComponentTypesView implements TreeDataProvider<ComponentType> {
 
     readonly odo = Odo.Instance;
     private registries: Registry[];
-    private readonly compDescriptions: Set<ComponentTypeDescription> = new Set<ComponentTypeDescription>();
-    public subject: Subject<void> = new Subject<void>();
```

Comment:
Once a devfiles list and/or a devfile is opened, it reacts only to the view's commands, so it you f.i. removed its Devfile Registry from the list of available registries, it will not be notified. And probably if you'll try to create a component from it - it'll fail.
However I haven't seen any cases which close the opened Devfile view when editing the list of Defvile registries.
I'll take a look on what we can do to restore this functionality.

Reply:
@datho7561  I restored this functionality, so now the Devfile Registry Editor and Create Component [from Template] lists  are updated on registry list changes.

However some more changes are required  in order to make it close the registries view and/or a devfile dialog view when a related devfile registry is removed.